util/util.go

```diff
@@ -220,7 +220,7 @@ func ExecuteWithTimeout(timeout time.Duration, binary string, args ...string) (s
 		return "", fmt.Errorf("Failed to execute: %v %v, output %s, stderr, %s, error %v",
 			binary, args, output.String(), stderr.String(), err)
 	}
-	return output.String(), nil
+	return strings.TrimSuffix(output.String(), "\n"), nil
```

Comment:
Why?

Reply:
In many places, the output from engine was written using fmt.Println("..."). So this result string would have "\n" character at the end. I noticed this while reading backupID. Hence, I'm using this to remove "\n" if at all present in the output.

Reply:
Will it affect the JSON parsing? It's easier for the cmdline output.